modules/dynamic/src/smithy4s/dynamic/internals/DynamicModelCompiler.scala

```diff
@@ -430,8 +432,8 @@ private[dynamic] object Compiler {
                 .map { case ((label, mShape), index) =>
                   val memberHints = allHints(mShape.traits)
                   schema(mShape.target)
-                    .map(_.addHints(memberHints: _*))
```

Comment:
does the order make a difference here?

Reply:
It very much does

Reply:
ahh yeah, in retrospec, my question did not make much sense